app/ui/src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts

```diff
@@ -108,31 +108,37 @@ export class DataMapperHostComponent extends FlowPage implements OnInit {
     // TODO not sure what to do for `none` or `any` here
     switch (kind) {
       case 'java':
-        this.addInitializationTask();
+
         const docDef: DocumentDefinition = this.cfg.addJavaDocument(
           type,
           isSource
         );
-        this.support.requestJavaInspection(connectorId, type).subscribe(
-          data => {
-            const inspection: string = data['_body'];
-            log.infoc(
-              () => 'Precomputed java document found for ' + type,
-              category
-            );
-            log.debugc(() => inspection, category);
-            docDef.initCfg.inspectionResultContents = inspection;
-            this.removeInitializationTask();
-          },
-          err => {
-            log.warnc(
-              () =>
-                'No precomputed java document found for ' + type + ': ' + err,
-              category
-            );
-            this.removeInitializationTask();
-          }
-        );
+
+        if ( specification != '' ) {
```

Comment:
I checked this yesterday and apparently out linter is skipping these constructs, although it's configured to block them. Basically  we do not allow extra spaces inside parens anymore in function or conditional expressions anymore.

Reply:
ms code marks it red without the spaces.

Reply:
It shouldn't. That seems to be caused by your visual style/linting config or perhaps you have the Angular Language Services plugin installed, which sometimes feature an erratic behavior when validating code.